frontend/src/lib/lemon-ui/ProfilePicture/ProfilePicture.tsx

```diff
@@ -51,7 +54,9 @@ export function ProfilePicture({
             />
         )
     } else {
-        pictureComponent = (
+        pictureComponent = isBot ? (
+            <IconRobot className={pictureClass} style={{ padding: '0.1rem' }} />
```

Comment:
0.1rem is 1.6px, which is pretty atypical
```suggestion
            <IconRobot className={pictureClass} className="p-0.5" />
````